public/apps/account/password-reset-panel.tsx

```diff
@@ -40,43 +40,35 @@ interface PasswordResetPanelProps {
 }
 
 export function PasswordResetPanel(props: PasswordResetPanelProps) {
-  const [currentPassword, setCurrentPassword] = useState<string>('');
+  const [currentPassword, setCurrentPassword] = React.useState<string>('');
```

Comment:
Since all useState has been replace, is the import become unused? Same comments applies for other places.

Reply:
Right. Removed useState from import statements.